program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+
+### PULP Code Base As-Is
+
+This is a research tool chain from ETHZ and U of Bologna which has the following attributes:
+- It is updated often as research projects need.
+- A precompiled version of that toolchain is available, compiled and tested from Embecosm, updated weekly.
+- Drawbacks are that it is out of date (based on earlier release of GCC), doesn't meet coding standards, and lacks testing
+
+
+
+### Standard RISC-V tool chain
+This is the standard RISC-V GCC tool chain whose attributes include:
+- The development is hosted by the FSF (Free software foundation). It is completely up to date with GCC and binutils. There are many commits per day.
+- It has no Pulp features
+- A precompiled version of that toolchain is available from Embecosm and possibly others. From Embecosm, the latest version is updated weekly. The latest table release is 10.2.0
+- It is currently used in the Ashling IDE product.
+
+### OpenHW CORE-V Reference Tool Chain
+This is a separate project in OpenHW managed within the SW TG. 
+- The project starts from the standard GCC tools (FSF) and adds CORE-V instructions
+- The binutils-gdb repository in OpenHW github currently has 2 branches
+ - Stable (currently, this repository is a mirror of FSF).
+ - Development (first commits to this repository will have support for CORE-V hardware loop)
+
+
+
+
+## List of project outputs
+
+*Need a list of outputs, i.e.*
+- *Documents to be produced*
+- *Downloadable code to be produced*
+
+
+## TGs Impacted/Resource requirements
+
+This work will be planned and discussed in the SW TG. The project plan will be produced by the project manager with assistance of SW TG, and reviewed/approved in TWG
+
+
+## OpenHW engineering staff resource plan: requirement and availability
+
+- Florian Zaruba - technical contribution and project planning
+- Duncan Bees - program management oversight
+
+## Engineering resource supplied by members - requirement and availability
+
+*The intent here is to write down specific time/person requirements. This is not yet available, and will be supplied in the project plan. As of Preliminary Project Proposal, the major gap identified is a resource for an initial application to test the IDE.*
+
+## OpenHW marketing resource - requirement and availability
+
+- Initial artwork suppplied by Publitek
+- No additional resource requirements yet identified
+
+## Marketing resource supplied by members - requirement and availability
+- No  resource requirements yet identified
+
+## Funding supplied by OpenHW - requirement and availability 
+- No funding requirements yet identified
+
+
+## Funding supplied by members - requirement and availability
+- No funding requirements yet identified
+
+
+
+## Architecture diagram
+*Need diagram showing SW architecture and also real-world physical setup for intended use. Action to Alexander* 

```

Comment:
I wonder if we are getting too bogged down here. Surely just a link to a standard Eclipse CDT diagram would suffice - we aren't doing anything non-standard.

Reply:
Alexander is planning to provide diagram that can be pointed to